StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -561,6 +561,18 @@ final class PartnerAuthViewController: UIViewController {
             navigateBack()
         } else {
             // we successfully launched the secure web browser
+            dataSource
+                .analyticsClient
+                .log(
+                    eventName: "auth_session.opened",
+                    parameters: [
+                        "browser": "ASWebAuthenticationSession",
```

Comment:
this parameter is mostly for Android; iOS just happens to always use the same "browser" (ASWebAuthenticationSession) rather than popping into an OS browser like Android